discord-service/src/main/java/pl/flyingoctopus/discord/member/action/AddMemberAction.java

```diff
@@ -65,15 +66,25 @@
             .numberOfArgs(ARGUMENTS_COUNT)
             .optionalArg(false)
             .desc("Sends xwiki invite to given email").build();
+    private static final Option TRELLO_TOKEN_OPT = Option.builder("token")
+            .required(false)
+            .longOpt("trello-token-api")
+            .hasArg()
+            .argName("TRELLO_TOKEN_API")
+            .numberOfArgs(ARGUMENTS_COUNT)
+            .optionalArg(false)
+            .desc("Used to add brief reports to user's trello card").build();
 
-    public static final List<Option> AVAILABLE_OPTIONS = List.of(USER_MENTION_OPT, MEMBER_EMAIL_OPT, TRELLO_EMAIL_OPT, WIKI_EMAIL_OPT);
+    public static final List<Option> AVAILABLE_OPTIONS = List.of(USER_MENTION_OPT, MEMBER_EMAIL_OPT, TRELLO_EMAIL_OPT, WIKI_EMAIL_OPT, TRELLO_TOKEN_OPT);
 
     private final EmailValidator emailValidator = EmailValidator.getInstance();
     private final HelpWithArgumentsAction helpAction = new HelpWithArgumentsAction("!fo member add", "Command used to add user as member", AVAILABLE_OPTIONS, "Read more here: https://wiki.flyingoctopus.pl/");
 
     private final MemberRepository memberRepository;
     private final ConfigurationService configurationService;
 
+    private final TrelloCardService trelloCardService;
```

Comment:
Możesz mi zostawić zapisanie tokena, jak ogarnę api trello to tutaj dodam odpowiedni kod